components/ui/checkbox.tsx

```diff
@@ -12,13 +16,40 @@ interface CheckboxProps {
   iconSize?: number;
 }
 
-const AnimatedCheck = Animated.createAnimatedComponent(Check);
+interface AnimatedCheckProps {
+  value: boolean;
+  size: number;
+  className: string;
+  opacity: number;
+}
+
+const AnimatedCheck = Animated.createAnimatedComponent(
+  React.forwardRef(
```

Comment:
I don't think a `forwardRef` would be necessary for this component

Reply:
I was basing this decision off the docs here:
https://docs.swmansion.com/react-native-reanimated/docs/core/createAnimatedComponent/#arguments

> "Function components have to be wrapped with React.forwardRef()."

It may be that we don't need a forward ref to solve this issue but in situations where you use a functional component, the docs recommended that. They may be out of date, of course

Reply:
My latest PR doesn't use any of this, so this should be resolved